doc/content/resources/codes_used.md

```diff
@@ -29,112 +29,112 @@ obtained through INL's [NCRC](https://inl.gov/ncrc/).
 
 ### Griffin
 
-- Molten Salt Fast Reactor [documentation](msr/msfr/griffin_pgh_model.md) [inputs](https://github.com/idaholab/virtual_test_bed/tree/main/msr/msfr/steady)
+- Molten Salt Fast Reactor [documentation](msr/msfr/griffin_pgh_model.md) and [inputs](https://github.com/idaholab/virtual_test_bed/tree/main/msr/msfr/steady)
```

Comment:
Without this `and`, the two links are kind of like one link in the page can can be quite confusing.